docs/site/content/docs/getting-started/run-kubernetes-cluster.md

```diff
@@ -14,81 +14,51 @@ toc = true
 top = false
 +++
 
-# Requirements
-
-Before using the theme, you need to install the [Zola](https://www.getzola.org/documentation/getting-started/installation/) ≥ 0.13.0.
-
-## Run the Theme Directly
-
-```bash
-git clone https://github.com/aaranxu/adidoks.git
-cd adidoks
-zola serve
-```
-
-Visit `http://127.0.0.1:1111/` in the browser.
-
-## Installation
-
-Just earlier we showed you how to run the theme directly. Now we start to
-install the theme in an existing site step by step.
-
-### Step 1: Create a new zola site
-
-```bash
-zola init mysite
-```
-
-### Step 2: Install AdiDoks
-
-Download this theme to your themes directory:
-
-```bash
-cd mysite/themes
-git clone https://github.com/aaranxu/adidoks.git
+# sealer run
+> Using a build image, you can run the run command to quickly create a cluster. Similar to the apply command, you can run the run command to deploy a cluster.
+
+Execute command parsing:
+
+```shell
+$ sealer run Kubernetes :v1.19.9 # Start cluster using Clusterfile configuration in image
```

Comment:
I think using examples to run a cluster is better, those content can be put them into Reference dir.

And we can separate different driver mods into different markdowns.

These docs focus on how to run Kubernetes cluster on BAREMETAL mod, how to add and delete a node, how to clean the cluster.